packages/instrumentation-openai/src/instrumentation.ts

```diff
@@ -520,4 +545,49 @@ export class OpenAIInstrumentation extends InstrumentationBase<any> {
       ? this._config.traceContent
       : true;
   }
+
+  private _addLogProbsEvent(
+    span: Span,
+    logprobs:
+      | ChatCompletion.Choice.Logprobs
+      | ChatCompletionChunk.Choice.Logprobs
+      | CompletionChoice.Logprobs
+      | null,
+  ) {
+    let result: { token: string; logprob: number }[] = [];
+
+    if (!logprobs) {
+      return;
+    }
+
+    if ("content" in logprobs) {
+      if (logprobs.content) {
+        result = logprobs.content?.map((logprob) => {
+          return {
+            token: logprob.token,
+            logprob: logprob.logprob,
+          };
+        });
+      }
+    } else if ("token_logprobs" in logprobs) {
```

Comment:
Here you can defintely just do a single `if ((logprobs as CompletionChoice.Logprobs).tokens)`